index.html

```diff
@@ -12,11 +12,28 @@
   <div id="app"></div>
   <script type="text/babel">
     class App extends React.Component {
-      render(){
+      state = {
+              name: 'Esri',
+              age: 20,
+          }
+          handleMouseOver(e){
+              console.log(e.target, e.pageX);
+          }
+          handleCopy(e){
+              console.log('Try being original for once');
+          }
+          handleClick(e){
+              console.log(e.target);
+          }
+        
+          render(){          
```

Comment:
Hmm, indentation is nie reg nie, ek sal jou nog wys hoe om linting op te stel in VSCode. 
Maar vir nou dink ek ook dis nog bietjie vroeg, laat ons eers by create-react-app video kom.